### ISSUE TITLE
Is this in the right place?

### ISSUE THREAD

**Issue by haleybarlar** [learn-co-curriculum/collections_practice_vol_2#24]
At the end of the Enumerables section, there's a lab called "Collections Practice 2".  It covers iteration, arrays, hashes and nested data structures, but at this point only iterations and arrays have been covered.  When I open the lab, it says I'm trying to access a lesson beyond my current lesson.  This would make sense, since the section after this lab is the introduction to hashes. Also theres a "Collections Practice" [the first one] way later on down the list.

I posted a question in the chat and someone said I should do it and that there was a reason for it being in this order.  So...is this in the right order? And if so, why does this include a few things that haven't been introduced?

**Comment by corypavitt1** [comment]
I agree.  I need a little more baby steps before getting to this point.  I'm getting completely lost.
